README.md

```diff
@@ -4,15 +4,14 @@
 ![license](https://img.shields.io/badge/License-Apache%202.0-blue?logo=apache&style=flat-square)
 [![maintainability](https://img.shields.io/codeclimate/maintainability-percentage/polkadot-js/apps?logo=code-climate&style=flat-square)](https://codeclimate.com/github/polkadot-js/apps)
 
-# @polkadot/apps
+# azero.dev
 
-A Portal into the Polkadot and Substrate networks. Provides a view and interaction layer from a browser.
+A portal containing advanced tooling for the Aleph Zero users looking for more capabilities than a regular signer can provide, like RPC calls, multisig, metadata update etc.
 
-This can be accessed as a hosted application via https://polkadot.js.org/apps/ or you can access the IPFS hosted version via https://polkadot.js.org/apps/ipfs (via hash) or https://dotapps.io (via ipns) to explore any of the supported Polkadot and Substrate chains.
-
-If you run one or more IPFS node(s), pinning the UI (which only gets updated on releases) will make it faster for you and others. You can find details about that below in the IPFS chapter below.
-
-**Important** If you are a chain developer and would like to add support for your chain to the UI, all the local configuration (API types, settings, logos) can be customized in [the apps-config package](packages/apps-config#README.md), complete with instructions of what goes where.
+The portal can be accessed through:
+* https://azero.dev/
```

Comment:
Can we please specify which environment it is for each url ie
```
## Mainnet
https://azero.dev/
## Testnet
https://test.azero.dev/
## Devnet
https://dev.azero.dev/
```

Reply:
Sure, I'll just use inline text to not mess the composition with titles, if you don't mind.